src/psyclone/tests/dynamo0p3_test.py

```diff
@@ -57,7 +57,17 @@
 # constants
 BASE_PATH = os.path.join(os.path.dirname(os.path.abspath(__file__)),
                          "test_files", "dynamo0p3")
+    # Get the root directory of this PSyclone distribution
+ROOT_PATH = os.path.dirname(os.path.dirname(os.path.dirname(
+    os.path.dirname(os.path.abspath(__file__)))))
+# Construct the path to the default configuration file
+DEFAULT_CFG_FILE = os.path.join(ROOT_PATH, "config", "psyclone.cfg")
+
+TEST_API = "dynamo0.3"
 
+# Our configuration object
+from psyclone.configuration import ConfigFactory
+_CONFIG = ConfigFactory().create()
```

Comment:
Should this be here? Actually, looking at the code, the question has turned into - will there be any side effects when running tests in parallel due to having this here, rather than a local create per test (even though it should be the same instance). If not, please ignore.

Reply:
No, there should be no side effects because I'm careful to only monkeypatch this Config object. I do create local Config objects in some of the tests when I want to initialise them with duff data.

Reply:
Different tests in parallel each have their own python process and therefore their own Config object.